.github/workflows/plugins.yml

```
@@ -1023,16 +1023,44 @@ jobs:
     runs-on: ubuntu-latest
     services:
       redis:
-        image: redis:4.0-alpine
+        image: redis:6.2-alpine
         ports:
           - 6379:6379
     env:
-      PLUGINS: redis|ioredis # TODO: move ioredis to its own job
+      PLUGINS: redis
       SERVICES: redis
     steps:
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
       - uses: ./.github/actions/plugins/test
 
+  ioredis:
+    runs-on: ubuntu-latest
+    services:
+      redis:
+        image: redis:6.2-alpine
+        ports:
+          - 6379:6379
+    env:
+      PLUGINS: ioredis
+      SERVICES: ioredis
```

I think this should be 
```suggestion
      SERVICES: redis
```

---

I think it's actually nice to have it as separate CI job. That way we can see if a single implementation fails or not :)